controllers/rhmi/bootstrapReconciler.go

```diff
@@ -299,22 +299,14 @@ func (r *Reconciler) deleteObsoleteService(ctx context.Context, serverClient k8s
 }
 
 func (r *Reconciler) setTenantMetrics(ctx context.Context, serverClient k8sclient.Client) error {
-	total, err := userHelper.GetTotalAPIManagementTenantsCount(ctx, serverClient)
-	if err != nil {
-		return err
-	}
-	reconciled, err := userHelper.GetReconciledAPIManagementTenantsCount(ctx, serverClient)
-	if err != nil {
-		return err
-	}
-	failed, err := userHelper.GetFailedAPIManagementTenantsCount(ctx, serverClient)
+	tenants := &integreatlyv1alpha1.APIManagementTenantList{}
+	err := serverClient.List(ctx, tenants)
```

Comment:
I wonder, would this be better in apimanagement tenant controller? What is the reason for it being in the bootstrap controller?

Reply:
After discussing this offline we decided to keep the implementation as is and we can move it the ApiMangementTenant controller later if needed.